### PR TITLE
[#199][#204][#210][#212] Fix bugs from PE-D

### LINKED ISSUE #199: [PE-D] Cannot completely fullscreen the application
As can be seen in the screenshot attached, upon clicking on the "maximise" button, next to the cross, their is a strip of my destop screen that had not been covered. And since it's in full screen, it cannot be moved to the centre of the screen either- it is rather awkwardly shifted to the left

Minimised view:

-------------
Labels: `severity.VeryLow` `type.FeatureFlaw`
original: kxshxsh/ped#1

### LINKED ISSUE #204: [PE-D] Using the Edit command.
After editing the name of a person, the display box did not get updated to the newly edited person's details. 

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: jiale-c/ped#2

### LINKED ISSUE #210: [PE-D] Command box is hidden when the screen is minimum
When a user has a small screen, they might wan to resize the application as small as possible. However, I was not able to use the command box in such scenario.

To replicate:
1. resize the window to the smallest size possible

Result:
User will not be able to see the command box and thus causing inconvenience to them.

Expected Result:
The command box should be available as such

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: clement0010/ped#1

### LINKED ISSUE #212: [PE-D] Bugs on UG
the link for Back to Table of Contents do not work. 

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: jiale-c/ped#5

### PR DESCRIPTION
**Pull request by lovemathboy**
Fixes #199 
Fixes #204 
Fixes #210 
Fixes #212

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -263,15 +263,17 @@ Format: `deletetag INDEX t/TAG…`
 * If any of the tag to be deleted does not exist, the command will be rejected.
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
- To overwrite all existing tags or remove all tags in one go, refer to [4.1.2. Editing a person](#412-editing-a-person-edit).
+
+To overwrite all existing tags or remove all tags in one go, refer to [4.1.2. Editing a person](#412-editing-a-person-edit).
+
```
Comment (atmh) [inline docs/UserGuide.md:268]: I don't think the extra line breaks are necessary 😄

File: docs/UserGuide.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ AddressBook pi (Abπ) is a **360° all-rounded desktop app for managing contacts
 
 <div style="page-break-after: always;"></div>
 
-## Table of Contents <br/>
+## Table of Contents
```
Comment (atmh) [inline docs/UserGuide.md:12]: Is the formatting still okay if there is no br tag after Table of Contents?

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 3]
```diff
@@ -54,6 +54,7 @@
 
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
+    public static final String MESSAGE_NO_CHANGE = "Nothing will change.";
```
Comment (atmh) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:57]: Could incorporate the word "(case-insensitive)" into the message to be more explicit, as `John Doe` equals to `john doe`.

File: src/main/resources/view/MainWindow.fxml [hunk 4]
```diff
@@ -13,7 +13,7 @@
 <?import javafx.scene.layout.VBox?>
 <?import javafx.stage.Stage?>
 
-<fx:root maxHeight="800.0" maxWidth="1200.0" minHeight="700.0" minWidth="800.0" onCloseRequest="#handleExit" title="AddressBook π" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
+<fx:root minHeight="600.0" minWidth="800.0" onCloseRequest="#handleExit" title="AddressBook π" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (atmh) [inline src/main/resources/view/MainWindow.fxml:16]: There is a warning about javaFX version 17, could you change it to version 11 for all the fxml files 👍 

image

**Review by atmh — CHANGES_REQUESTED** [review]
LGTM, just some nits to fix 👍

File: src/main/resources/view/MainWindow.fxml (hunk 4 above)
Comment (lovemathboy) [reply src/main/resources/view/MainWindow.fxml]: Changed!

File: src/main/java/seedu/address/logic/commands/EditCommand.java (hunk 3 above)
Comment (lovemathboy) [reply src/main/java/seedu/address/logic/commands/EditCommand.java:57]: Modified so the error message only shows when the matching is case-sensitive.

File: docs/UserGuide.md (hunk 2 above)
Comment (lovemathboy) [reply docs/UserGuide.md:12]: Yep!

File: docs/UserGuide.md (hunk 1 above)
Comment (lovemathboy) [reply docs/UserGuide.md]: Removed.

**Review by atmh — APPROVED** [review]
LGTM

**Pull request merged**